build.sbt

```diff
@@ -693,6 +693,7 @@ lazy val tools = project
       "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion,
       "org.apache.avro" % "avro" % avroVersion % Provided,
       "org.apache.parquet" % "parquet-hadoop" % parquetVersion,
+      "org.apache.hadoop" % "hadoop-common" % hadoopVersion,
```

Comment:
I'm on the fence about relying so heavily on the hadoop `Configuration` class, from the hadoop-common artifact. Parquet is trying to move away from `Configuration` and onto their own [ParquetConfiguration](https://github.com/apache/parquet-java/blob/master/parquet-common/src/main/java/org/apache/parquet/conf/ParquetConfiguration.java) class, which we could use instead. However, it might be confusing for Scio users since Scio is heavily dependent on `Configuration` and we don't have immediate plans to offboard from it

Reply:
Actually I might pull this out into a separate PR. will update shortly